CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Qiita changelog
 
-
-Qiita 0.1.0-dev (changes since Qiita 0.1.0 go here)
+Qiita 0.2.0-dev (changes since Qiita 0.2.0 go here)
 ---------------------------------------------------
 
+Qiita 0.2.0
```

Comment:
Can you change this to read `Version 0.2.0 (2015-08-25`? See the previous title for an example.